extensions/micrometer/deployment/src/main/java/io/quarkus/micrometer/deployment/export/PrometheusRegistryProcessor.java

```diff
@@ -13,6 +13,7 @@
 import io.quarkus.micrometer.runtime.MicrometerRecorder;
 import io.quarkus.micrometer.runtime.config.MicrometerConfig;
 import io.quarkus.micrometer.runtime.config.PrometheusConfigGroup;
+import io.quarkus.micrometer.runtime.export.PrometheusMeterRegistryProducer;
```

Comment:
Do not import this directly. The Prometheus registry classes may not be around (it is optional).
```suggestion

```

Reply:
Hm, I don't get it. Previously, the `io.quarkus.micrometer.runtime.export.PrometheusMeterRegistryProvider` was imported and so it was exactly the same situation. In other words, we should not import `PrometheusMeterRegistryProvider` either... for the same reason.